cli/azd/cmd/testdata/TestUsage-azd.snap

```diff
@@ -7,6 +7,7 @@ Usage
 Commands
   Configure and develop your app
     auth     	: Authenticate with Azure.
+    build    	: Builds the application's code. (Beta)
```

Comment:
Why (Beta)?

Reply:
It's new(ish) and may change.  We have an alpha, beta, GA strategy but we aren't applying it very consistently at the moment.  Open for any feedback here.